react-native-libraries.json

```diff
@@ -12664,5 +12664,17 @@
     "ios": true,
     "android": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/JUHUU-Labs/sdk-ts",
+    "npmPkg": "@juhuu/sdk-ts",
+    "ios": true,
+    "android": true,
+    "web": true,
+    "expoGo": true,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
```

Comment:
Hey @MoritzWallner, thank you for adding the new entry!

Just a nit before we can merge, let's remove the `false` values:
```suggestion
```